### PR TITLE
Return Exit Code 1 for failled commands

### PR DESCRIPTION
**Pull request by geof90**
This helps with CLI automation.

### CODE REVIEW

File: cli/script/cli.ts [hunk 1]
```diff
@@ -12,7 +12,10 @@ function run(): void {
     }
 
     execute(parser.command)
-        .catch((error: any): void => console.error(chalk.red("[Error]  " + error.message)))
+        .catch((error: any): void => {
+            console.error(chalk.red("[Error]  " + error.message))
```
Comment (dtivel) [inline cli/script/cli.ts]: ;

File: cli/script/command-executor.ts [hunk 2]
```diff
@@ -531,7 +525,7 @@ function logout(): Promise<void> {
             });
     }
 
-    return Q(<void>null);
+    return Q.fcall(() => { throw new Error("You are already logged in from this machine."); });
```
Comment (dtivel) [inline cli/script/command-executor.ts]: Is this error message contextually correct?  (Copy and paste error?)

**Comment by silhouettes** [comment]
Nice! LGTM

**Comment by dtivel** [comment]
looks good

**Pull request merged**